<launch>

  <node pkg="gpup_gp_node_exp" name="gp_transition" type="gp_transition.py" output="screen"/>
  <!-- <node pkg="gpup_gp_node_exp" name="gpup_transition" type="gpup_transition.py" output="screen"/> -->

</launch>